api-gateway-config/conf.d/marathon_apis.conf

```diff
@@ -1,191 +0,0 @@
-#/*
```

Comment:
Ok to remove it as default, but let’s update README and document how to configure the GW w Marathon. Most of this is already written up, just needs some reorg.